RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECALUncorrected_cfi.py

```diff
@@ -38,8 +38,8 @@
     algoName = cms.string("LocalMaximumSeedFinder"),
     thresholdsByDetector = cms.VPSet(
     cms.PSet( detector = cms.string("ECAL_ENDCAP"),
-              seedingThreshold = cms.double(0.6),
-              seedingThresholdPt = cms.double(0.15)
+              seedingThreshold = cms.double(0.50),
+              seedingThresholdPt = cms.double(0.50)
```

Comment:
do we have any Z->ee plots to show that this increase is OK?
I can imagine that shower or brem cases can degrade quite a bit.
@Sam-Harper what do you think?

Reply:
So we discussed this at length in the ECAL+E/gamma meeting with jetMET. JetMET need these thresholds to control the MET. For E/gamma, they are likely not ideal (although  the results could be surprising with PU contributions), for the reasons you say. The plan is to check if it really causes a major (and it has to be major) problem for E/gamma although given the timeline, they may not complete in time for pre4 in which case we'll just go ahead.  

If 10_0_0 has slightly worse performance in the endcap at low pt, its no big deal, we'll fix it next release. The samples we will need in 10_0_0 will be for energy regressions and we save the raw for those so in a pinch we can always re-reco them ourselves with new settings.